Cargo.toml

```diff
@@ -46,6 +46,7 @@ tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 uts2ts = "0.4.1"
 path-clean = "1.0.1"
+fs_extra = "1.3.0"
```

Comment:
Do we require this dependency? We can simply add a function for traversing the directory and getting the size, which is essentially what `get_size` is doing: <https://docs.rs/fs_extra/latest/src/fs_extra/dir.rs.html#786-816>